README.md

```diff
@@ -128,6 +128,19 @@ GPU memory.
 Select the version of the TensorFlow library to be used, available
 versions are 1 and 2. Default version is 1.
 
+##### --backend-config=tensorflow,default-max-batch-size=\<int\>
+
+The value to use for max_batch_size during [Autocomplete](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#auto-generated-model-configuration) when no 
+other value is found. If the `--strict-model-config=false` command-line
+option is used, the tensorflow backend will set the max_batch_size
+of the model to this default value under the following conditions:
+
+1. Autocomplete has determined the model is capable of batching requests. 
+2. max_batch_size is 0 in the model configuration or max_batch_size 
+   is omitted from the model configuration.
+
+If max_batch_size > 1 and no [scheduler](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#scheduling-and-batching) is provided, the dynamic batch scheduler will be used.
```

Comment:
Move  `is provided, the dynamic batch scheduler will be used.` entirely to a new line. We try to break very long lines into multiple.
Also the new statement can be `dynamic batch scheduler will be enabled.`

Reply:
Fixed.